SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -140,22 +151,40 @@ - (void)disconnect {
 
 #pragma mark - Creating Session Streams
 
-- (void)sdl_establishSession {
+- (BOOL)sdl_tryConnectAccessory:(EAAccessory *)accessory {
```

Comment:
I believe this is consistent with the Apple style guidelines for APIs that attempt to perform an operation and return NO if failure occurs. An example is tryLock in NSLock.